src/modules/setup/setupTypings.ts

```diff
@@ -104,26 +118,35 @@ export const setupTypings = async (
   ignoreLinked: boolean,
   buildersWithTypes = BUILDERS_WITH_TYPES
 ) => {
+  log.info('Setup typings')
```

Comment:
missed this one :sweat_smile:

```suggestion
  log.info('Setting up typings')
```